data/json/items/ammo/20x66mm.json

```diff
@@ -102,7 +102,7 @@
     "id": "20x66_inc",
     "copy-from": "20x66_shot",
     "type": "AMMO",
-    "name": { "str_sp": "20x66mm incendiary" },
+    "name": { "str": "20x66mm incendiary", "str_pl": "20x66mm incendiaries" },
```

Comment:
Other incendiary rounds such as `7.62x51mm incendiary, reloaded` don't have a different plural, so it might be better to keep this change or change the other rounds to have a different plural for consistency.

Reply:
I keep going back and forth on this one.

Reply:
Went back to "str_sp" and "incendiary".